packages/backend-api/src/api/services/updateNotifications.ts

```diff
@@ -53,10 +51,12 @@ const userFields = ['id', 'name', 'email', 'avatarURL', 'group', 'isActive'];
 const commonFields = ['id', 'updatedAt', 'allCount', 'unprocessedCount', 'unmoderatedCount', 'moderatedCount',
   'approvedCount', 'highlightedCount', 'rejectedCount', 'deferredCount', 'flaggedCount',
   'batchedCount', 'recommendedCount', 'assignedModerators', ];
-const categoryFields = [...commonFields, 'label'];
+const categoryFields = [...commonFields, 'label', 'ownerId'];
 const articleFields = [...commonFields, 'title', 'url', 'categoryId', 'sourceCreatedAt', 'lastModeratedAt',
   'isCommentingEnabled', 'isAutoModerated'];
 
+const idFields = new Set(['categoryId', 'tagId', 'ownerId']);
```

Comment:
I think it's slightly better style to have global constants as all-caps, what do you think?